<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CarMaintenance</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>2</y>
      <width>181</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Car Maintenance</string>
    </property>
   </widget>
   <widget class="QLabel" name="notesLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>251</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="dateLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date of Maintenace</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>94</y>
      <width>252</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="maintenanceComboBox">
       <item>
        <property name="text">
         <string>Oil Change</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coolant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Brake Fluid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Brake Replacement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Engine Air Filter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wiper Replacement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tire Quality Check</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tire Replacement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tire Rotation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tire Pressure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MaintenanceLabel">
       <property name="text">
        <string>Car Maintenance</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>214</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Entry</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>0</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Records</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
